### ISSUE TITLE
Lint/EmptyConditionalBody Error

### ISSUE THREAD

**Issue by baarkerlounger** [rubocop/rubocop#10534]
rubocop throws an error on `Lint/EmptyConditionalBody`

--------

## Expected behavior

Rubocop parses the file and does not throw an error

## Actual behavior
```
1 error occurred:
An error occurred while Lint/EmptyConditionalBody cop was inspecting file.rb
```

## Steps to reproduce the problem

Code block throwing the error:

```ruby
map_row(row).each do |attr_key, attr_val|
          update = case_log.update(attr_key => attr_val)
          unless update
            # TODO: determine what to do
          end
        rescue ArgumentError
          # TODO: determine what we want to do
        end
```

## RuboCop version

1.27. This was error was not being thrown on 1.25.

```
1.27.0 (using Parser 3.1.2.0, rubocop-ast 1.17.0, running on ruby 3.1.1 x86_64-linux)
  - rubocop-performance 1.13.3
  - rubocop-rails 2.13.2
  - rubocop-rake 0.6.0
  - rubocop-rspec 2.9.0
```

**Comment by koic** [comment]
This issue is a dup with #10514 and resolved by #10516.
